cmake/Platform/Mac/InstallUtils_mac.cmake.in

```diff
@@ -0,0 +1,168 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+function(fixup_qt_framework lib_name framework_path)
+
+    file(REMOVE_RECURSE
+        ${framework_path}/Headers
+        ${framework_path}/Resources
+        ${framework_path}/${lib_name}
+        ${framework_path}/Versions/Current
+        ${framework_path}/Versions/5/Headers
+    )
+
+    execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink 5 Current
+        WORKING_DIRECTORY ${framework_path}/Versions
+    )
+    execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink Versions/Current/${lib_name} ${lib_name}
+        WORKING_DIRECTORY ${framework_path}
+    )
+    execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink Versions/Current/Resources Resources
+        WORKING_DIRECTORY ${framework_path}
+    )
+
+endfunction()
+
+function(fixup_python_framework framework_path)
+
+    file(REMOVE_RECURSE
+        ${framework_path}/Versions/Current
+        ${framework_path}/Versions/3.7/Headers
+        ${framework_path}/Versions/3.7/lib/Python
+        ${framework_path}/Versions/3.7/lib/python3.7/test
+        ${framework_path}/Versions/3.7/lib/python3.7/site-packages/scipy/io/tests
+        ${framework_path}/Python
+        ${framework_path}/Resources
+        ${framework_path}/Headers
+    )
+    
+    file(GLOB_RECURSE exe_file_list "${framework_path}/**/*.exe")
+    if(exe_file_list)
+        file(REMOVE_RECURSE ${exe_file_list})
+    endif()
+    execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink include/python3.7m Headers
+        WORKING_DIRECTORY ${framework_path}/Versions/3.7
+    )
+    execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink 3.7 Current
+        WORKING_DIRECTORY ${framework_path}/Versions/
```

Comment:
Can we place the version number of 3.7 into a variable at the top of this file or better yet set the version of `3.7` in the [cmake/LYPython.cmake](https://github.com/o3de/o3de/blob/development/cmake/LYPython.cmake) into a variable within that script and read that value here?

Reply:
LYPython.cmake already has a python version set. But it includes the patch version as well. I can add a new variable there for just major minor as well.

Reply:
I added LY_PYTHON_VERSION_MAJOR_MINOR and set it to 3.7